back/engines/commercial/analysis/app/controllers/analysis/web_api/v1/questions_controller.rb

```diff
@@ -4,6 +4,7 @@ module Analysis
   module WebApi
     module V1
       class QuestionsController < ApplicationController
+        include FilterParamsExtraction
```

Comment:
Now using the same mixin as the `SummarizationController` instead of its own implemenation, which diverged from the right one in the mixin. I thought this was already the case, clearly an oversight when refactoring previously.

Reply:
Really hard to add a proper test for this, since we'd either have to create 100s of ideas to see a difference in filter behavior, or mock a lot of stuff. So decided to leave it as-is.